src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -300,6 +301,18 @@ def load_command_table(self, _):
             supports_no_wait=True)
         g.wait_command('wait')
 
+    backup_short_term_retention_policies_operations = CliCommandType(
+        operations_tmpl='azure.mgmt.sql.operations#BackupShortTermRetentionPoliciesOperations.{}',
+        client_factory=get_sql_backup_short_term_retention_policies_operations)
+
+    with self.command_group('sql db str-policy',
+                            backup_short_term_retention_policies_operations,
+                            client_factory=get_sql_backup_short_term_retention_policies_operations,
+                            is_preview=True) as g:
+
+        g.custom_command('set', 'update_short_term_retention', supports_no_wait=True)
```

Comment:
I am thinking if we should use `update` as command name? May I know the operation is Patch or PUT?

Reply:
The swagger itself has both PUT and Patch

Reply:
with support_no_wait, a wait command will be expected to add